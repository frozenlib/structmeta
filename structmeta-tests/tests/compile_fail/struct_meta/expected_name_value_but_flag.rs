#[structmeta_tests::attr_name_value(value)]
fn my_func() {}

fn main() {}
