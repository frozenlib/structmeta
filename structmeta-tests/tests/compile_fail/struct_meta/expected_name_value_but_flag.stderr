error[E0433]: failed to resolve: could not find `attr_name_value` in `structmeta_tests`
 --> $DIR/expected_name_value_but_flag.rs:1:21
  |
1 | #[structmeta_tests::attr_name_value(value)]
  |                     ^^^^^^^^^^^^^^^ could not find `attr_name_value` in `structmeta_tests`
